support/spring/src/main/java/org/apache/shiro/spring/web/config/AbstractShiroWebConfiguration.java

```diff
@@ -123,6 +131,19 @@ protected Cookie buildCookie(String name, int maxAge, String path, String domain
         cookie.setPath(path);
         cookie.setDomain(domain);
         cookie.setSecure(secure);
+        cookie.setSameSite(Cookie.SameSiteOptions.LAX);
```

Comment:
nit: you could call the newer method from here

Reply:
I did't run the test because merging the ClassUtilsTest blocked mvn compile… I was a bit quick. I'll add a cleanup commit first to the 1.6.x branch (via PR).